.gitattributes

```diff
@@ -0,0 +1 @@
+* text eol=lf
```

Comment:
The contents of this file should be: `* text=auto`

This allows folks to use whatever line endings are configured on their OS, but when they get checked into git they'll be standardized to lf. Let me know if that doesn't seem to be working for you!